yt_dlp/extractor/archiveorg.py

```diff
@@ -945,3 +946,238 @@ def _real_extract(self, url):
+        if max_video is not None:
+            video_url = self._WAYBACK_BASE_URL + max_video.get('source')
+            video_req = self._request_webpage(HEADRequest(video_url), video_id, note='Check video availablity', errnote=False, fatal=False)
+            if video_req:
+                formats.append({
+                    'url': video_url
+                })
+
+        # Code from NaverBaseIE
+        automatic_captions = {}
+        subtitles = {}
+        for caption in traverse_obj(vod_data, ('captions', 'list'), []):
+            caption_url = caption.get('source')
+            if not caption_url:
+                continue
+            caption_url = self._WAYBACK_BASE_URL + caption_url
+            sub_dict = automatic_captions if caption.get('type') == 'auto' else subtitles
+            lang = caption.get('locale') or join_nonempty('language', 'country', from_dict=caption) or 'und'
+            if caption.get('type') == 'fan':
+                lang += '_fan%d' % next(i for i in itertools.count(1) if f'{lang}_fan{i}' not in sub_dict)
+            sub_dict.setdefault(lang, []).append({
+                'url': caption_url,
+                'name': join_nonempty('label', 'fanName', from_dict=caption, delim=' - '),
+            })
+
+        return {
+            'id': video_id,
+            'title': traverse_obj(player_info, ('postDetail', 'post', 'officialVideo', 'title')),
+            'formats': formats,
+            'subtitles': subtitles,
+            'automatic_captions': automatic_captions,
+            'uploader_id': traverse_obj(vod_data, ('meta', 'user', 'id')),
+            'uploader': traverse_obj(vod_data, ('meta', 'user', 'name')),
+            'uploader_url': traverse_obj(vod_data, ('meta', 'user', 'url')),
+            'creator': traverse_obj(player_info, ('postDetail', 'post', 'author', 'nickname')),
+            'channel': traverse_obj(player_info, ('postDetail', 'post', 'channel', 'channelName')),
+            'channel_id': traverse_obj(player_info, ('postDetail', 'post', 'channel', 'channelCode')),
+            'duration': int_or_none(traverse_obj(player_info, ('postDetail', 'post', 'officialVideo', 'playTime'))),
+            'view_count': int_or_none(traverse_obj(player_info, ('postDetail', 'post', 'officialVideo', 'playCount'))),
+            'like_count': int_or_none(traverse_obj(player_info, ('postDetail', 'post', 'officialVideo', 'likeCount'))),
+            'comment_count': int_or_none(traverse_obj(player_info, ('postDetail', 'post', 'officialVideo', 'commentCount'))),
+            'timestamp': int_or_none(traverse_obj(player_info, ('postDetail', 'post', 'officialVideo', 'createdAt')), scale=1000),
+            'release_timestamp': int_or_none(traverse_obj(player_info, ('postDetail', 'post', 'officialVideo', 'willStartAt')), scale=1000),
+            'thumbnail': traverse_obj(vod_data, ('meta', 'cover', 'source')),
+        }
```

Comment:
Maybe something like this could be cleaner
```suggestion
return {
    'id': video_id,
    'formats': formats,
    'subtitles': subtitles,
    'automatic_captions': automatic_captions,
    **traverse_obj(player_info, ('postDetail', 'post', {
        'title': ('officialVideo', 'title'),
        'creator': ('author', 'nickname'),
        'channel': ('channel', 'channelName'),
        'channel_id': ('channel', 'channelCode'),
        'duration': ('officialVideo', 'playTime', {int_or_none}),
        'view_count': ('officialVideo', 'playCount', {int_or_none}),
        'like_count': ('officialVideo', 'likeCount', {int_or_none}),
        'comment_count': ('officialVideo', 'commentCount', {int_or_none}),
        'timestamp': ('officialVideo', 'createdAt', {lambda x: int_or_none(x, scale=1000)}),
        'release_timestamp': ('officialVideo', 'willStartAt', {lambda x: int_or_none(x, scale=1000)}),
    })),
    **traverse_obj(vod_data, ('meta', {
        'uploader_id': ('user', 'id'),
        'uploader': ('user', 'name'),
        'uploader_url': ('user', 'url'),
        'thumbnail': ('cover', 'source'),
    }))
}
```